sections/image-with-text.liquid

```diff
@@ -242,7 +242,7 @@
       "type": "color_scheme",
       "id": "section_color_scheme",
       "label": "t:sections.all.colors.label",
-      "default": "background-1"
+      "default": "scheme-1"
```

Comment:
Seeing a some stray background-1 default values (in 7 files I believe), presumably from recent changes that were brought in with the rebase. For example, just below here in image-with-text, there's now another color-scheme type setting that needs this same change.

Reply:
Done! I also double-checked that no additional background-2, accent-1, accent-2 and inverse appeared after the rebase!